pkg/cmd/target/target.go

```diff
@@ -100,15 +106,16 @@ func runCmdTarget(f util.Factory, o *TargetOptions) error {
 type TargetKind string
 
 const (
-	TargetKindGarden  TargetKind = "garden"
-	TargetKindProject TargetKind = "project"
-	TargetKindSeed    TargetKind = "seed"
-	TargetKindShoot   TargetKind = "shoot"
-	TargetKindPattern TargetKind = "pattern"
+	TargetKindGarden       TargetKind = "garden"
+	TargetKindProject      TargetKind = "project"
+	TargetKindSeed         TargetKind = "seed"
+	TargetKindShoot        TargetKind = "shoot"
+	TargetKindPattern      TargetKind = "pattern"
```

Comment:
We have a TargetKind "pattern" in the enum but the value is not a valid TargetKind?

Reply:
What do you mean? Right now, when you do not provide garden, project, seed, shoot or control-plane. It defaults to pattern. We need this now when we introduce subcommands we can get rid of the targetkinds and pattern targeting will be used for the root command and the subcommands do the other target kinds.
Anyway this has nothing to do with this PR.

Reply:
Sorry. I have missed the fact that pattern of course is in the ValidTargetKinds list. I don't knwo anymore why I thought it is invalid to call `gardenctl target pattern ...`